src/libarchive/libarchive.zig

```diff
@@ -510,7 +506,8 @@ pub const Archive = struct {
                         }
                     }
 
-                    var tokenizer = std.mem.tokenize(u8, bun.asByteSlice(pathname), std.fs.path.sep_str);
+                    // TODO(dylan-conway): I think this should only be '/'
+                    var tokenizer = std.mem.tokenizeAny(u8, bun.asByteSlice(pathname), "/\\");
```

Comment:
you're probably right since this is parsing of (cross platform) tarball files.
we shouldnt do tokenizeAny on posix though, i think it is a little slower.